habitat-baselines/habitat_baselines/rl/hrl/hl/fixed_policy.py

```diff
@@ -37,8 +37,10 @@ def _parse_solution_actions(self, solution):
                 hl_action.name,
                 [x.name for x in hl_action.param_values],
             )
-            # Filter out the correct action for the robot.
+            # Filter out the correct action for the robot
             robot_id = "robot_" + self._agent_name.split("_")[1]
+            if robot_id == "robot_agent":
```

Comment:
What is this for? Why do we have to deal with this special case?

Reply:
It seems that in the config yaml, the agent is called "robot_agent", but the pddl solution yaml is called "robot_0". Maybe we want to have two solution yamls one for single agent, and the other one for multi-agent?

Reply:
Yes, I think the better thing would be to have two solution yamls.